cmd/wallet-sdk-gomobile/api/api.go

```diff
@@ -7,6 +7,20 @@ SPDX-License-Identifier: Apache-2.0
 // Package api defines gomobile-compatible wallet-sdk interfaces.
 package api
 
+// JSONObject contains a single JSON object (not an array).
+// It's a simple wrapper around the actual JSON string. Its purpose is to help the
+// caller using the mobile bindings to understand what type of data to expect or pass in.
+type JSONObject struct {
+	JSON string
```

Comment:
so we decided to have string here instead of bytes?

bytes are ideal way to pass large data.

Reply:
rename `JSON` to `data`, struct name already implies that it is JSON

Reply:
Renamed

Reply:
This can help differentiate it from something key bytes, which truly are raw bytes

Reply:
One reason I named it `JSON` instead of `Data` is because I didn't want the caller to confuse the struct itself with the actual JSON string... although I'm not sure if there's a super clean way to do that.

Reply:
Changed back to []byte for now